README.md

```diff
@@ -7,6 +7,26 @@ used in conjunction with external log rotation.
 
 **Note** The following of file renames does not work on Windows, because `File.Stat.inode` is used to determine whether the log file has been (re)moved and, on non-Unix, `File.Stat.inode` is always 0.
 
+## Getting Started
+
+There are some brief examples on usage below, but I highly recommend you review the
+API docs [here](https://hexdocs.pm/timex), there are many examples, and some extra pages with
```

Comment:
Good call. I was more than a little hasty.

> On Jun 28, 2016, at 11:19 AM, Kurt Williams <notifications@github.com> wrote:
> 
> In README.md:
> 
> > @@ -7,6 +7,26 @@ used in conjunction with external log rotation.
> >  
> >  **Note** The following of file renames does not work on Windows, because `File.Stat.inode` is used to determine whether the log file has been (re)moved and, on non-Unix, `File.Stat.inode` is always 0.
> >  
> > +## Getting Started
> > +
> > +There are some brief examples on usage below, but I highly recommend you review the
> > +API docs [here](https://hexdocs.pm/timex), there are many examples, and some extra pages with
> You'll need to update this link.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>